src/api/middleware/utils.js

```diff
@@ -6,6 +6,7 @@ const lodash = require("lodash");
 const DEFAULT_ERROR_CODE = ErrorTypes.VALIDATION_ERROR;
 const DEFAULT_ERROR_MSG = ValidationReasons.UNKNOWN;
 const DEFAULT_STATUS_CODE = HTTPStatus.BAD_REQUEST;
+const MAX_FILE_SIZE = 10; // MB
```

Comment:
Always clarify units on constants, the comment won't be visible when using the constant in other files, you can remove it.

```suggestion
const MAX_FILE_SIZE_MB = 10;
```